src/kernels/errors/kernelErrorHandler.ts

```diff
@@ -249,37 +249,38 @@ export abstract class DataScienceErrorHandler implements IDataScienceErrorHandle
         error: RemoteJupyterServerConnectionError,
         errorContext?: KernelAction
     ) {
-        const info = await this.serverUriStorage.get(error.serverId);
+        const info = await this.serverUriStorage.get(error.serverHandle);
```

Comment:
Consistently use Serverhandle everwhere instead of generating an id called `serverId` and passing that around.